crossgovernmentservices/csd_notes
# Enable adding tags to notes on creation or update

## What
- Added Tag model and migration
- Updated add note and edit note views to enable adding tags
- Updated list notes template to display note tags
## How to review
1. Run app as per README instructions
2. Browse to [the notes page](http://localhost:5000/notes)
3. Write a note and add some comma-separated tags
4. Click "Save"
5. See the note in the notes list with the associated tags listed inline
6. Click the note to edit
7. Edit the tags
8. Click "Save"
9. See the note in the notes list with updated tags
## Who can review

Not @andyhd

Fixes https://trello.com/c/BQ8yayQ3
